src/main/ruby/core/truffle/io_operations.rb

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+
+module Truffle
+  module IOOperations
+    def self.last_line(a_binding)
+      Truffle::KernelOperations.frame_local_variable_get(:$_, a_binding)
+    end
+
+    def self.set_last_line(value, a_binding)
+      Truffle::KernelOperations.frame_local_variable_set(:$_, a_binding, value)
+    end
+
+    Truffle::Graal.always_split(method(:last_line))
+    Truffle::Graal.always_split(method(:set_last_line))
```

Comment:
Please keep `always_split` calls right below the method, just like `private`, etc.

Reply:
@aardvark179 Can you fix (and the case from the previous PR) in another PR?